Tests/hBridge.py

```diff
@@ -0,0 +1,11 @@
+from ..Modules.Magnetorquer.hBridge import hBridge
+
+actuator = hBridge(23, 24, 14, 15)
+
+while True:
+    inp = input('Enter direction : ')
```

Comment:
Can we actually take input on the satellite side?

Reply:
It's a test, it's not intended for use in the experiments only for testing each module.